src/components/forms/FamilyForm.tsx

```diff
@@ -945,11 +972,21 @@ export const FamilyForm = ({ family: loadedFamily }: TProps) => {
               )}
             </VStack>
 
-            <ButtonGroup isDisabled={!canModify(watchOrganisation, userAccess)}>
+            <ButtonGroup
+              isDisabled={
+                loadedFamily
+                  ? !canModify(
```

Comment:
We are calling the same method here with 2/3 of the same arguments, for readability could we change to something like: `!canModify(loadedFamily?.organisation || orgName, isSuperuser, userAccess)` ?

Reply:
I didn't realise you could do that! Super cool, doing now